java-test-samples/apigw-lambda-list-s3-buckets/metadata.json

```diff
@@ -0,0 +1,46 @@
+{
+    "title": "API Gateway with Lambda and S3",
+    "description": "This project contains unit, and integration tests for a pattern using API Gateway, AWS Lambda and Amazon S3.",
```

Comment:
Is this the "starter project"? We can label it as such?

Reply:
Yes, Updated the title accordingly